runtime/src/transaction_priority_details.rs

```diff
@@ -25,6 +25,8 @@ pub trait GetTransactionPriorityDetails {
         _round_compute_unit_price_enabled: bool,
     ) -> Option<TransactionPriorityDetails> {
         let mut feature_set = FeatureSet::default();
+        // this feature was enabled on mainnet-beta since epooch 537
+        // https://github.com/solana-labs/solana/issues/30366
         feature_set.activate(
```

Comment:
what's the point of activating this. All branches which would depend on this being activted in `process_compute_budget_instructions` are removed in this PR, right?

Reply:
right, rebase and cleaned it up at [9d607cb](https://github.com/solana-labs/solana/pull/34208/commits/9d607cbe3616cddc765d309b0314d41b60a7b6e6)